feature: Move Email self-hosted configuration to file (#372)
codacy/chart: docs/configuration/integrations/email.md

# Emails

> ## SMTP Configuration
> 
> Follow the instructions below to set up the Codacy integration with SMTP to send emails:

Reviewer: Simplify to avoid having a subsection:

```suggestion
Follow the instructions below to set up Codacy to send emails using your SMTP server:
```

> 1.  Edit the file `values-production.yaml`, set `global.email.enabled: "true"` and define the remaining values with the credentials for your smtp server:

Reviewer: ```suggestion
1.  Edit the file `values-production.yaml`, set `global.email.enabled: "true"` and define the remaining values with the credentials for your SMTP server:
```

> 1.  Edit the file `values-production.yaml`, set `global.email.enabled: "true"` and define the remaining values with the credentials for your smtp server:

Reviewer: Is the YAML block supposed to be already included in the `values-production.yaml` file? :thinking: It seems it's not there:

https://github.com/codacy/chart/blob/move-email-config-to-file/codacy/values-production.yaml

Author: forgot that part

    ```yaml
    email:
        enabled: "true"
        replyTo: "notifications@mycompany.com"
        smtp:
            hostname: "smtp.mycompany.com"
            protocol: "smtps" # smtps | smtp
            username: "my-smtp-username"

>             password: "a-s!r0ng-pVsSwORG"

Reviewer: Can we have comments to guide the user setting this up in the same style as we have for other configurations in the `values-production.yaml`, for example https://github.com/codacy/chart/blob/master/codacy/values-production.yaml#L114 ?
